inference-engine/tests/functional/plugin/shared/include/behavior/ov_core_integration.hpp

```diff
@@ -478,30 +478,11 @@ TEST_P(OVClassImportExportTestP, smoke_ImportNetworkNoThrowWithDeviceName) {
     SKIP_IF_CURRENT_TEST_IS_DISABLED()
     ov::runtime::Core ie = createCoreWithTemplate();
     std::stringstream strm;
-    ExecutableNetwork executableNetwork;
+    ov::runtime::ExecutableNetwork executableNetwork;
     ASSERT_NO_THROW(executableNetwork = ie.compile_model(actualNetwork, deviceName));
-    ASSERT_NO_THROW(executableNetwork.Export(strm));
+    ASSERT_NO_THROW(executableNetwork.export_model(strm));
     ASSERT_NO_THROW(executableNetwork = ie.import_model(strm, deviceName));
-    ASSERT_NO_THROW(executableNetwork.CreateInferRequest());
-}
-
-TEST_P(OVClassImportExportTestP, smoke_ExportUsingFileNameImportFromStreamNoThrowWithDeviceName) {
```

Comment:
why have we removed this test?

Reply:
`ExecutableNetwork::Export(const std::string&);` was removed?

Reply:
but we can modify the tests with `ExecutableNetwork::Export(std::ostream&);` ?

Reply:
`smoke_ImportNetworkNoThrowWithDeviceName` does the same but just uses an in-memory byte stream instead of a file.